Makefile

```diff
@@ -206,6 +203,10 @@ otelcorecol:
 genotelcorecol:
 	pushd cmd/builder/ && $(GOCMD) run ./ --skip-compilation --config ../otelcorecol/builder-config.yaml --output-path ../otelcorecol && popd
 
+.PHONY: ocb
+ocb:
+	$(MAKE) -C cmd/builder ocb
```

Comment:
Where is the binary being generated? At the root of the repository, or at ./bin ?

Reply:
Based on the builder's make file it gets output in `./bin` right?

Reply:
Yup, it does the same as `otelcorecol`, just as a developer convenience.